src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { FC, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './Types/Types';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function findUser(userId: number) {
+  return usersFromServer.find(({ id }) => id === userId);
+}
+
+function getTodos(todos: Todo[]) {
+  return todos.map(todo => {
+    const cloneTodo = { ...todo };
+
+    cloneTodo.user = findUser(cloneTodo.userId);
+
+    return cloneTodo;
+  });
```

Comment:
instead of function, you can use a variable `todos`  to get current todos

```
export const todos: Todo[] = todosFromServer.map(todo => ({
  ...todo,
  user: getUser(todo.userId),
}));
```